### DISCUSSION TITLE
Optionally Remove embedded "View URL" Link in Alert messages

### DISCUSSION THREAD

**Discussion by blackswan1** [grafana/grafana#67615] [Alerting]
In some cases, due to security restrictions, it's not possible to access the Grafana Alert Link supplied with the Alert message. e.g. MS-Teams WebHook.
Hence the Button/Link is inoperable.

It would be a big plus to get the choice whether the link shall be displayed/embedded in the alert message or not.

From the community:
https://community.grafana.com/t/remove-grafana-alert-url-from-ms-teams-alert-notification/85433

**Comment by nctuevamax** [comment]
It's also what I want.

**Comment by JDA88** [comment]
I also agree, the link is not functional for us on most notification canal.
We need an option to remove it or beter a way to change the "template" that generate it.

**Comment by tomasrojas26** [comment]
I'm also interested in this solution, it would be better if we can customize the link in that button.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #83018. Feel free to continue the discussion around this there. Thank you!
